eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -177,7 +177,19 @@ function New-ChangeLogEntry {
     return $null
   }
 
-  if (!$Content) { $Content = @() }
+  if (!$Content) { 
+    $Content = @()
+    $Content += ""
+    $Content += "### Features Added"
+    $Content += ""
+    $Content += "### Breaking Changes"
+    $Content += ""
+    $Content += "### Key Bugs Fixed"
+    $Content += ""
+    $Content += "### Fixed"
+    $Content += ""
+    $Content += ""
```

Comment:
Do we want 2 empty lines here?

Reply:
Might be worth having a look at what one of these look when added to the changelog.

Reply:
Here is now it'll look
```
# Release History

## 1.15.0 (Unreleased)

### Features Added

### Breaking Changes

### Key Bugs Fixed

### Fixed


## 1.15.0-beta.1 (Unreleased)

### Features Added

- Types to represent `GeoJson` primitives.
```